### DISCUSSION TITLE
recipes/ingredients/candied-orange-peel/

### DISCUSSION THREAD

**Discussion by giscus** [nicholaswilde/recipes#633] [Comments] [answered]
# recipes/ingredients/candied-orange-peel/

A collection of my recipes.

https://nicholaswilde.io/recipes/ingredients/candied-orange-peel/

**Comment by nicholaswilde** [comment, marked as answer]
Did this with lemon peel instead of orange using a vegetable peeler. They might have come out a bit thin. Should use a knife to peel the fruit next time to get a bit thicker and more of the pith.
